apps/editing-toolkit/editing-toolkit-plugin/block-patterns/class-block-patterns-from-api.php

```diff
@@ -117,27 +117,35 @@ function ( $a, $b ) {
 					$viewport_width = $viewport_width < 320 ? 320 : $viewport_width;
 					$pattern_name   = self::PATTERN_NAMESPACE . $pattern['name'];
 					$block_types    = $this->utils->maybe_get_pattern_block_types_from_pattern_meta( $pattern );
+					$pattern_props  = array(
+						'title'         => $pattern['title'],
+						'description'   => $pattern['description'],
+						'content'       => $pattern['html'],
+						'viewportWidth' => $viewport_width,
+						'categories'    => array_keys(
+							$pattern['categories']
+						),
+						'isPremium'     => $is_premium,
+						'blockTypes'    => $block_types,
+					);
+
+					$post_types = $this->utils->get_pattern_post_types_from_pattern( $pattern_props );
+					if ( $post_types ) {
+						$pattern_props['postTypes'] = $post_types;
+					}
```

Comment:
Nitpick: Since we're running `update_pattern_post_types` now on all registered patterns, I think we can drop this piece of code.